editoast/src/views/mod.rs

```diff
@@ -49,6 +52,41 @@ pub fn study_routes() -> impl HttpServiceFactory {
     ]
 }
 
+#[derive(OpenApi)]
+#[openapi(
+    info(description = "My Api description"),
+    tags(),
+    paths(),
+    components(schemas(), responses())
+)]
+pub struct OpenApiRoot;
+
+impl OpenApiRoot {
+    pub fn build_openapi() -> serde_json::Value {
+        let manual = include_str!("../../openapi.yaml").to_owned();
+        let mut openapi = OpenApiRoot::openapi();
+        // Remove the operation_id that defaults to the endpoint function name
+        // so that it doesn't override the RTK methods names.
+        for (_, endpoint) in openapi.paths.paths.iter_mut() {
+            for (_, operation) in endpoint.operations.iter_mut() {
+                operation.operation_id = None;
+                // By default utoipa adds a tag "crate" to operations that don't have
+                // any. That causes problems with RTK tag management.
+                match &operation.tags {
+                    Some(tags) if tags.len() == 1 && tags.get(0).unwrap() == "crate" => {
+                        operation.tags = None;
+                    }
+                    _ => (),
+                }
+            }
+        }
+        let generated = openapi
+            .to_json()
+            .expect("the openapi should generate properly");
+        OpenApiMerger::new(manual, generated).finish()
+    }
+}
+
```

Comment:
can you move this stuff to a separate module?

Reply:
The `OpenApiRoot` is supposed to aggregate all `views` endpoints and schemas, so it makes sense that it's in mod.rs. 

For the last line of the function is supposed to be updated each time an endpoint is annotated and that its generated operations are supposed to replace the manually written ones. So it's convenient to have it just below `OpenApiRoot`.

Thus I'm not really feeling like creating a new module just for a for loop, wdt?

Reply:
But we can definitely plan a bit of refactoring when all the endpoints are annotated and the old openapi.yml is dropped. I'm just saying it's more convenient for the annotating process, which is already tedious enough